src/soc/intel/meteorlake/include/soc/iomap.h

```diff
@@ -66,9 +66,10 @@
 #define IOE_P2SB_BAR		IOE_PCR_ABOVE_4G_BASE_ADDR
 #define IOE_P2SB_SIZE		(256 * MiB)
 
-#define IOM_BASE_ADDR		0x3fff0aa0000
+/* IOE_P2SB_BAR + 0xaa0000, but iasl refuses to perform arithmetics  */
```

Comment:
What? Can you point to the faulting code?

Reply:
Same exact bug as https://review.coreboot.org/c/coreboot/+/84216

Reply:
Perhaps it's worth adding such references (if known) to the commit description

Reply:
Right, amended: https://github.com/Dasharo/coreboot/pull/609/commits/d047598fdf4e9ce66ed26225d03ea9abf957f1d2